### PR TITLE
feat(client): inform sfu about pause and resume of tracks

### PR DESCRIPTION
**Pull request by santhoshvai**

### CODE REVIEW

File: packages/client/src/devices/CameraManager.ts [hunk 1]
```diff
@@ -56,18 +56,14 @@ export class CameraManager extends InputMediaDeviceManager<CameraManagerState> {
   /**
    * Disables the video tracks of the camera
    */
-  pause() {
-    this.state.mediaStream?.getVideoTracks().forEach((track) => {
-      track.enabled = false;
-    });
+  pause(): Promise<void> {
+    return this.call.pauseTrack(TrackType.VIDEO);
```
Comment (szuperaz) [inline packages/client/src/devices/CameraManager.ts:60]: This won't work on the lobby screen

**Comment by szuperaz** [comment]
We'll have API differences with Android, their `pause` and `resume` methods behave differently. How about this: 
- `disable` and `pause` methods have a param where we can specify if we want to stop or disable the track
- `resume` should work as in Android: it resumes (starts/enables) the media track if it was previously enabled

File: packages/client/src/devices/CameraManager.ts (hunk 1 above)
Comment (santhoshvai) [reply packages/client/src/devices/CameraManager.ts:60]: yes I see.. so the way is to get the stream here and pause the tracks.. but send to sfu once joined then

**Comment by santhoshvai** [comment]
closing this PR since we will remove pause and resume and to move to this API below 

```
enable()
disable({ stop }). -- by default stop is false
```

**Pull request closed without merging**